include/aktualizr-lite/tuf/aklite-tuf.h

```diff
@@ -0,0 +1,248 @@
+#ifndef AKLITE_TUF_API_H_
```

Comment:
I would inline it with the path, e.g. `AKLITE_TUF_TUF_H_` -> `aktualizr-lite/tuf/tuf.h` (AKLITE is a short form of `aktualizr-lite`).